docs/it/testing.md

```diff
@@ -167,6 +167,19 @@ Feature: Google Searching
     Then results for "panda" are shown
 ```
 
+### Mutation Testing
+
+Il mutation testing è una metodologia di test in cui il codice sorgente viene modificato in modo da introdurre volutamente degli errori. Questi errori vengono chiamati _mutanti_. L'obiettivo del mutation testing è di verificare che i test siano in grado di rilevare questi mutanti. Se un mutante non viene rilevato da un test, viene considerato _vivo_ e i test a riguardo vengono considerati non sufficienti. Se un mutante viene rilevato da un test, viene considerato _morto_ e i test a riguardo vengono considerati sufficienti per quel mutante.
```

Comment:
Considerazioni:
- la parola "mutanti" potrebbe non essere corretta in questo caso: il codice subisce delle mutazioni (introduzione degli errori). Il "mutante" è la versione mutata, non gli errori intromessi.
- ho messo all'inizio il motivo del mutation test in modo da rendere chiaro il motivo per cui esistono.


```suggestion
Il mutation testing è una metodologia di test per valutare l'attuale test suite. Partendo dal codice, si creano diverse versioni diverse, ognuna delle quali introduce una mutazione, piccola a piacere. Queste versioni mutate vengono chiamati _mutanti_. L'obiettivo del mutation testing è di verificare se la test suite identifica la mutazione: se un mutante non viene rilevato da un test, viene considerato _vivo_ e i test a riguardo vengono considerati non sufficienti per quella mutazione. Se un mutante viene rilevato da un test, viene considerato _morto_ e i test a riguardo vengono considerati sufficienti per quel mutante.
```